### DISCUSSION TITLE
How to convert slate object to HTML and render it?

### DISCUSSION THREAD

**Discussion by nedvoid** [ianstormtaylor/slate#4774] [Q&A] [answered]
I have the following retrieved from the database and I'm trying to render it as HTML ( not inside an editor ).
```
[{"type":"paragraph","children":[{"text":"some random user content !#€@"}]}]
```
I tried installing `slate-html-serializer` but that seems to have been removed in the latest version and no longer supported.

**What options do I have?**

**Comment by StormVanDerPol** [comment, marked as answer]
You can write a serializer.
the docs can point you in this direction:
https://docs.slatejs.org/concepts/10-serializing

**Comment by thompsonsj** [comment]
When writing a serializer for HTML, I found it tricky - lots to consider such as whitespace, how to handle `code` tags, converting back to Slate (that last one is for my use case!)...etc.

Actively working on a package that hopefully makes this easier: https://github.com/thompsonsj/slate-serializers.
